api/client/samples/crop_models/sao_paulo_sugar_content/Sao_Paulo_brazil_sugar_content_modeling.ipynb

```diff
@@ -0,0 +1,3628 @@
+{
```

Comment:
<p>the aggregation for pet and rainfall should be *sum* not mean</p><p> </p><p>technically it should give the same results since it's just multiplied by the number of days. but the values makes more sense as a the sum </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>Done</p>